cmake/Toolchains/arm_cortex_a72_hardfp_native.cmake

```diff
@@ -4,5 +4,6 @@
 ########################################################################
 set(CMAKE_CXX_COMPILER g++)
 set(CMAKE_C_COMPILER  gcc)
-set(CMAKE_CXX_FLAGS "-march=armv8-a -mtune=cortex-a72 -mfpu=neon-fp-armv8 -mfloat-abi=hard" CACHE STRING "" FORCE)
+set(CMAKE_CXX_FLAGS "-ffast-math -march=armv8-a -mtune=cortex-a72 -mfpu=neon-fp-armv8 -mfloat-abi=hard" CACHE STRING "" FORCE)
```

Comment:
Recently we had a discussion on `-ffast-math` and my understanding is that we should not use it because it breaks things.

Reply:
Yes if I remember correctly we agreed that it *might* break things because it allows non IEEE-754 compliant optimizations, and is therefore not a good default.

On the other hand NEONv7 is also not strictly IEEE-754 compliant and all tests passed for me with fast-math enabled.

But for now I think it's better to leave it out!